ui/Modal/ManagePeers.svelte

```diff
@@ -92,7 +92,7 @@
           dataCy="peer-input"
           hint="v"
           bind:value={newPeer}
-          placeholder="Paste a remote address here"
+          placeholder="Paste a peer address here..."
```

Comment:
Are we using the word "address"? Or should it be `peerID` -- or whatever casing we're going for?

Reply:
> > Add tooltip to Manage Remotes modal?
> 
> Is it these tooltips?
> <img alt="Screenshot 2020-11-09 at 09 37 47" width="50%" src="https://user-images.githubusercontent.com/158411/98518136-45a3a980-226f-11eb-88e0-2700f9742a8d.png">
> 
> If so, I had them there initially, but [had to remove them](https://github.com/radicle-dev/radicle-upstream/pull/1155/files#diff-52c26d24276defc75692d1883dd7f53ffdde278c62b6c6a5ce0fd15c4e904e75L57) after a discussion with @radicle-dev/product.

Ok, good to know. This was something I added so it could be discussed. Seems like we can make the Manage Remotes flow easier to understand with a tooltip, but I'll check out the previous conversation on it. thnx

Reply:
It now [says](https://github.com/radicle-dev/radicle-upstream/pull/1206/files#diff-cda325a03becd89efda336f88f202071175a39195079244f352ddb39bf04e535R95): `Paste a Device ID here…`.